linera-storage-service/tests/store_test.rs

```diff
@@ -2,7 +2,7 @@
 // SPDX-License-Identifier: Apache-2.0
 
 use linera_storage_service::{
-    child::{get_free_port, StorageService},
+    child::{get_free_endpoint, StorageService},
```

Comment:
Since this is a large PR, it would help to split changes like this into a separate one.

Reply:
Ok, done.